radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/forks/ForkBuilder.java

```diff
@@ -64,51 +64,64 @@
 
 package com.radixdlt.statecomputer.forks;
 
-import com.google.common.hash.HashCode;
+import com.google.common.collect.ImmutableSet;
 import java.util.Optional;
 
 /**
  * An intermediate class used for creating fork configurations. Allows to modify various config
  * parameters before the actual fork config object is created.
  */
 public final class ForkBuilder {
+  public static record FixedEpochForkBuildersOpts(long epoch) {}
```

Comment:
Could you please explain what "BuildersOpts" mean?

Reply:
different options for the two types of fork configs (candidate and fixed epoch)

Reply:
the builder is the outer class, this is just a container for two different types of fork parameters